MH-12934, Update translations

lkiesow opened the pull request:
- Update the translations to the current state on Crowdin.
- Add new languages: Filipino, Tagalog, Turkish

lkiesow committed:
MH-12934, Update translations

- Update the translations to the current state on Crowdin.
- Add new languages: Filipino, Tagalog, Turkish

staubesv commented on modules/admin-ui/package-lock.json:
```diff
@@ -5,13 +5,13 @@
     "abbrev": {
       "version": "1.1.1",
-      "resolved": "https://nexus.opencast.org/nexus/content/repositories/npmjs/abbrev/-/abbrev-1.1.1.tgz",
+      "resolved": "http://nexus.opencast.org/nexus/content/repositories/npmjs/abbrev/-/abbrev-1.1.1.tgz",
```
Hi @lkiesow This change seems not related to updating translations. Why is it included in this PR?

staubesv reviewed (changes requested):
Hi @lkiesow 

I've found two issues:

- Theodul Player needs likely to be adapted to support the new languages (see https://bitbucket.org/opencast-community/opencast/pull-requests/1792/mh-12515-add-language-support-for/diff#Lmodules/matterhorn-engage-theodul-core/src/main/resources/ui/js/engage/core.jsT184, for example)
- Changes to package-lock.json seem not related to this PR

lkiesow committed:
MH-12934, Update languages supported by Theodul

This updates the list of languages supported by the Theodul player.

lkiesow commented:
- Updated list of languages supported by Theodul (this includes sorting the list and removing all languages no longer supported)
- removed `package-lock.json`

staubesv commented:
Hi @lkiesow Please note that I've merged this without waiting for CI to complete (was running for more than 2 hours).
